tron/actioncommand.py

```diff
@@ -41,48 +27,58 @@ class ActionCommand(object):
       done      (when the command is finished)
     """
 
-    COMPLETE = ActionState('complete')
-    FAILSTART = ActionState('failstart')
-    EXITING = ActionState('exiting', close=COMPLETE)
-    RUNNING = ActionState('running', exit=EXITING)
-    PENDING = ActionState('pending', start=RUNNING, exit=FAILSTART)
+    PENDING = 'pending'
+    RUNNING = 'running'
+    EXITING = 'exiting'
+    COMPLETE = 'complete'
+    FAILSTART = 'failstart'
+
+    STATE_MACHINE = Machine(
+        PENDING,
+        **{
```

Comment:
I know what `**` does, but looking at this code I don't really understand it.
Are you sure normal kwargs wouldn't be more clear here?

Reply:
the problem with normal kwargs is that `PENDING=xyz` is interpreted as `"PENDING"=xyz` instead of `<value of PENDING>=xyz`